origin-js/src/resources/discovery.js

```diff
@@ -77,6 +78,17 @@ class Discovery {
 
     return this.query(query)
   }
+
+  async getListing(listingId) {
```

Comment:
What's the use for this method ? Currently the DApp gets listing data by calling marketplace.getListing - it might be simpler to keep it that way since then it works both in decentralized and centralize mode ?

Reply:
Thanks yes, this is no longer needed. Deleting it